package.json

```diff
@@ -39,6 +39,7 @@
     "@web/dev-server-import-maps": "^0.0.6",
     "@web/test-runner": "0.13.27",
     "@web/test-runner-commands": "0.6.1",
+    "@adobe/helix-rum-js": "1.6.0",
```

Comment:
If we just copy the code anyway, is there any benefit in adding this as a dev dependency?

Reply:
At the moment no benefit, it just shows that the file is a copy from `node-modules/@adobe/helix-rum-js` .
If as next step you would like use `rollup`  to include the file in your code, then it would be relevant.

Reply:
We use microbundle for preact, which might be an option. But IMO just copy/pasting seems to be the more straight forward way for now